src/standalone.js

```diff
@@ -78,6 +79,10 @@ export default function ({
     }
     if (syntax === "scss") {
       postcssProcessOptions.syntax = scssSyntax
+    } else if (syntax === "less") {
+      postcssProcessOptions.syntax = lessSyntax
+      postcssProcessOptions.mixinsAsAtRules = true
```

Comment:
I think instead of adding these options directly we should add a `syntaxOptions` option, which `mixinsAsAtRules` would be one property of. This object could then be reused by other syntaxes as needed.

This makes me think that we should probably add `syntax` and `syntaxOptions` to the configuration, because otherwise there is no way to specify syntax options from the CLI.